core/legacy/gather/gather-runner.js

```diff
@@ -424,37 +418,6 @@ class GatherRunner {
 
     const baseArtifacts = passContext.baseArtifacts;
 
-    // Fetch the manifest, if it exists.
-    try {
-      baseArtifacts.WebAppManifest = await WebAppManifest.getWebAppManifest(
-        passContext.driver.defaultSession, passContext.url);
-    } catch (err) {
-      log.error('GatherRunner WebAppManifest', err);
-      baseArtifacts.WebAppManifest = null;
-    }
-
-    try {
-      baseArtifacts.InstallabilityErrors = await InstallabilityErrors.getInstallabilityErrors(
-        passContext.driver.defaultSession);
-    } catch (err) {
-      log.error('GatherRunner InstallabilityErrors', err);
-      baseArtifacts.InstallabilityErrors = {
-        errors: [
-          {
-            errorId: 'protocol-timeout',
-            errorArguments: [],
-          },
-        ],
-      };
-    }
-
-    try {
-      baseArtifacts.Stacks = await Stacks.collectStacks(passContext.driver.executionContext);
-    } catch (err) {
-      log.error('GatherRunner Stacks', err);
-      baseArtifacts.Stacks = [];
```

Comment:
I'm not sure what happens now if this artifact errors. Probably similar to above, we need to try/catch and set a default in the gatherer such that `artifacts.Stacks` will never be undefined (or an Error).